gravityflow/gravityflow
# Ensure Notification Steps Apply Pre Submission Filter

## Description
The GPPA crew submitted a bug report noting that when they use a Workflow Notification Step (vs. a built-in Gravity Forms notification), fields display their value instead of their label. 

After some digging, it turns out this is due to the fact that we don't pass the form object through the `gform_pre_submission_filter` before evaluating it. As such, none of the filters GPPA applies to ensure labels are displayed actually wind up being applied.

## Testing instructions
1. Add a form with a single text field an a radio field.
2. Using GPPA, set the radio field to use dynamic choices. 
3. Preview the form, and submit it with the default Notification enabled. 
4. Check the email and ensure you see the Label for the value you chose in the Radio field.
5. Now edit the form settings and disable the default notification. 
6. Add a Workflow Notification step, and choose the default admin notification as the one to send. 
7. Submit the form again. 
8. Check the email and ensure you still see the Label being displayed, and not the value.

## Checklist:
